sirepo/package_data/static/json/madx-schema.json

```diff
@@ -472,6 +479,16 @@
             "sequence": ["BEAMLINE", "LatticeBeamlineList", "", "Name of the sequence to be loaded and expanded."],
             "period": ["BEAMLINE", "LatticeBeamlineList", "", "Name of the sequence to be loaded and expanded."]
         },
+        "initialTwissParams": {
+            "alpha": ["Alpha", "Float", 0.0],
+            "beta": ["Beta", "Float", 1.0],
+            "dim": ["Dimension", "SpaceCoordinate", "x"],
+            "eta": ["Emittance", "Float", 1.0],
+            "gamma": ["Gamma", "Float", 1.0]
+        },
+        "particleTracking": {
+            "numParticles": ["Number of particles", "Integer", 1000, "", 1, 100000]
```

Comment:
in "simulation" we have "numberOfParticles". These should probably be the same value. Doesn't matter to me which one we use.

Reply:
I now see there is also a beam parameter ```NPART``` which is particles per bunch (madx default is 0 for some reason). Does that serve the same purpose or is it it's own thing?

Reply:
Hmm I hadn't noticed that. I think maybe it is its own thing. We use the numberOfParticles to generate the `ptc_start` commands. NPART seems to be doing something else internal with madx. 

You can see the docs [here](https://mad.web.cern.ch/mad/webguide/manual.html#Ch7). They describe NPART as "the particles in the bunch". I'm not really sure how NPART and the ptc_start commands are related.